pyxis/pyxis.go

```diff
@@ -34,6 +34,10 @@ func registerMissingCves(tx *gorm.DB, apiImageCves []string) error {
 	toInsertCves := []models.Cve{}
 	var found bool
 	for _, cveName := range apiImageCves {
+		if !utils.IsValidCVE(cveName) {
+			logger.Info(fmt.Sprintf("Invalid CVE obtained from pyxis: %s, skipping", cveName))
```

Comment:
1. You don't need `fmt.Sprintf`, if you use `logger.Infof`
2. You are missing the skipping part :)

Reply:
3. Maybe Warn log level would be better

Reply:
omg